lib/killbill_client/models/payment_method.rb

```diff
@@ -41,6 +41,18 @@ def find_in_batches_by_search_key(search_key, offset = 0, limit = 100, options =
               options
         end
 
+        def find_by_external_key(external_key, plugin_property = nil, included_deleted = false, with_plugin_info = false, audit='NONE', options = {})
+          get "#{KILLBILL_API_PAYMENT_METHODS_PREFIX}",
+              {
+                  :externalKey => external_key,
+                  :pluginProperty => plugin_property,
```

Comment:
By convention, we pass the `pluginProperty` in the `options` hash instead. This is done automatically for all calls: https://github.com/killbill/killbill-client-ruby/blob/c7ab884d288d1e9037210ac9bed884db91670ef9/lib/killbill_client/api/net_http_adapter.rb#L39.